pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Context.java

```diff
@@ -262,7 +268,18 @@
      *                         "disableReplication"
      *
      * @return A future that completes when the framework is done publishing the message
+     * @deprecated in favor of using {@link #newOutputMessage(String, Schema)}
```

Comment:
this was added very recently. I believe it is not included in 2.3.x release. If so we should delete this interface as opposed to deprecating it

Reply:
@jerrypeng @sijie i am ok, what do you think about this?

Reply:
Good point, @srkukarni ! We can remove this method, since it was introduced in #4005 which has not been released yet.

Reply:
yup we can remove that

Reply:
ok will remove that